### PR TITLE
2024-07-18 Fees  Data

### PR DESCRIPTION
**Pull request by Tritium-VLK**
Not rounding this time around.  I don't think we need it anymore:

- Summary with the processed data sheet: https://docs.google.com/spreadsheets/d/1gpuy7kCg4t8ZmBkdrKPve9TocyjEcq0lPR3aMdYAW3A/edit?usp=sharing

- JSON: https://drive.google.com/file/d/1eIRVnc_sB8OTuAAfmdG3yYaLNTiNT84J/view?usp=sharing

### CODE REVIEW

File: fee_allocator/fees_collected/fees_2024-07-04_2024-07-18.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+{
+  "mainnet": 251328.80,
+  "arbitrum": 36140.14,
+  "polygon": 12540.96,
```
Comment (Xeonus) [inline fee_allocator/fees_collected/fees_2024-07-04_2024-07-18.json:4]: Raw amount for Poly is 15528223994 so should be 15528.22?

File: fee_allocator/fees_collected/fees_2024-07-04_2024-07-18.json [hunk 2]
```diff
@@ -0,0 +1,8 @@
+{
+  "mainnet": 251328.80,
```
Comment (Xeonus) [inline fee_allocator/fees_collected/fees_2024-07-04_2024-07-18.json:2]: You are rounding up here. It should be 251328.79

**Review by Xeonus — CHANGES_REQUESTED** [review]
Please check raw amounts and correctly round to those, seems you rounded up a few cents on totals which might cause issues

File: fee_allocator/fees_collected/fees_2024-07-04_2024-07-18.json (hunk 1 above)
Comment (Tritium-VLK) [reply fee_allocator/fees_collected/fees_2024-07-04_2024-07-18.json:4]: Note the comment at the top. I don't think this matters anymore after we refactored the aura redistribution.

**Review by Xeonus — APPROVED** [review]
lgtm now. thanks ser

**Pull request merged**